app/src/androidTest/java/org/systers/mentorship/LoginActivityTest.kt

```diff
@@ -132,6 +133,7 @@ class LoginActivityTest {
      * with:
      * username: EMPTY
      * password: PRESENT
```

Comment:
In many places, comments aren't correct if you can change it might be helpful. However PR LGTM. Here password: EMPTY

Reply:
I think the password is filled in this test. The doc is for ` fun testLoginButtonClickedWhenUsernameIsEmptyAndPasswordIsFilled()` afaik

Reply:
Yeah, you are right absolutely I was reading the doc for the upper test instead it was for below. I am sorry.

Reply:
@aditmehta9 thank you for paying attention to detail in your review 🙌 🙌